classes/Visualizer/Gutenberg/src/Components/ChartSelect.js

```diff
@@ -32,13 +32,70 @@ const { startCase } = lodash;
 
 const { __ } = wp.i18n;
 
+const {
+	Button
+} = wp.components;
+
 const {
 	Component,
 	Fragment
 } = wp.element;
 
 const { InspectorControls } = wp.blockEditor || wp.editor;
 
+let vm, vmv;
+vm = visualizer.media = {};
+vmv = vm.view = {};
```

Comment:
We can use names like `visualizerMediaView` to improve readability. We not have to keep the awful names from `library.js`